MacOS-10/BlueBubblesHelper/BlueBubblesHelper.m

```diff
@@ -436,6 +439,60 @@ -(void) handleMessage: (NetworkController*)controller  message:(NSString *)messa
             index++;
         }
         [BlueBubblesHelper sendMessage:(data) transfers:[transfers copy] attributedString:attributedString transaction:(transaction)];
+    } else if ([event isEqualToString:@"vetted-aliases"]) {
+
+        NSArray* aliasNames = [BlueBubblesHelper getVettedAliases];
+
+        if (transaction != nil) {
+            [[NetworkController sharedInstance] sendMessage: @{@"transactionId": transaction, @"identifier": aliasNames}];
+        }
+
+    } else if ([event isEqualToString:@"register-alias-listener"]) {
+
+        [BlueBubblesHelper registerListenerForActiveAliasChanged];
+
+        if (transaction != nil) {
+            [[NetworkController sharedInstance] sendMessage: @{@"transactionId": transaction}];
+        }
+
+    } else if ([event isEqualToString:@"deactivate-alias"]) {
```

Comment:
I think we should change this to just auto-register, maybe it can attach itself when we send the notification to the server that we are connected (right in the spot where we have the "uncomment to test" code)

Reply:
Agreed